CONTRIBUTING.md

```diff
@@ -146,6 +146,12 @@ factor](https://en.wikipedia.org/wiki/Bus_factor) on the project,
 disseminating information about contributions and changes in order to
 prevent creation of a bottleneck.
 
+### Technical writing
+
+Please follow our [technical writing style
+guide](./docs/standalone/writing-style-guide.md) when
+contributing changes to documentation.
```

Comment:
Should: move this further up

We have already an entry about updating the documentation on line 20 and this document is kind of structured from basic to advanced. Following the style guide for writing markdown is quite basic :)